[2022-06-16 00:14:51,628] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: say_hi.team_2 manual__2022-06-16T00:14:45.313823+00:00 [queued]>
[2022-06-16 00:14:51,648] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: say_hi.team_2 manual__2022-06-16T00:14:45.313823+00:00 [queued]>
[2022-06-16 00:14:51,648] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-16 00:14:51,648] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2022-06-16 00:14:51,649] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-16 00:14:51,695] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): team_2> on 2022-06-16 00:14:45.313823+00:00
[2022-06-16 00:14:51,709] {standard_task_runner.py:52} INFO - Started process 2851 to run task
[2022-06-16 00:14:51,737] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'say_hi', 'team_2', 'manual__2022-06-16T00:14:45.313823+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/username.py', '--cfg-path', '/tmp/tmpxpa32jtc', '--error-file', '/tmp/tmpvlqsmjom']
[2022-06-16 00:14:51,737] {standard_task_runner.py:80} INFO - Job 37: Subtask team_2
[2022-06-16 00:14:52,188] {task_command.py:370} INFO - Running <TaskInstance: say_hi.team_2 manual__2022-06-16T00:14:45.313823+00:00 [running]> on host 6a14fd44e8dc
[2022-06-16 00:14:52,697] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=say_hi
AIRFLOW_CTX_TASK_ID=team_2
AIRFLOW_CTX_EXECUTION_DATE=2022-06-16T00:14:45.313823+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-16T00:14:45.313823+00:00
[2022-06-16 00:14:52,699] {logging_mixin.py:115} INFO - Porto
[2022-06-16 00:14:52,699] {python.py:173} INFO - Done. Returned value was: None
[2022-06-16 00:14:52,815] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=say_hi, task_id=team_2, execution_date=20220616T001445, start_date=20220616T001451, end_date=20220616T001452
[2022-06-16 00:14:52,889] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-16 00:14:53,155] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
